### PR TITLE
feat: Made a guide in Compass instrument to show how to calibrate the axis

### LINKED ISSUE #1273: No guide to see which axis is parallel to ground in Compass instrument
**Actual Behaviour**

There is no guide to see which axis is parallel to the ground and thus selecting proper radio button becomes difficult for a new user

**Expected Behaviour**

There should be a guide with a screenshot showing all the three axis in the mobile

**Steps to reproduce it**

Download apk from PR #1271 -> Open `Compass` instrument -> Analyse

**LogCat for the issue**

Not applicable

**Screenshots of the issue**

Not applicable

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by harsh-2711**
Fixes #1273 

**Changes**: Added a help icon in the toolbar to show the guide on how to calibrate compass with the axis

**Screenshot/s for the changes**: 

**Checklist**: [Please tick following check boxes with `[x]` if the respective task is completed]
- [x] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding them
- [x] No modifications done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`
- [x] I have reformatted code in every file included in this PR [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>]
- [x] My code does not contain any extra lines or extra spaces
- [x] I have requested reviews from other members

**APK for testing**: 
[app-debug.zip](https://github.com/fossasia/pslab-android/files/2222729/app-debug.zip)

### CODE REVIEW

**Comment by harsh-2711** [comment]
@cweitat @woshikie @CloudyPadmal @abhinavraj23 @Avjeet Please review.

File: app/src/main/java/org/fossasia/pslab/activity/AxisDetectionHelpActivity.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
+package org.fossasia.pslab.activity;
+
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v7.app.AppCompatActivity;
+
+import org.fossasia.pslab.R;
+
+public class AxisDetectionHelpActivity extends AppCompatActivity {
```
Comment (CloudyPadmal) [inline app/src/main/java/org/fossasia/pslab/activity/AxisDetectionHelpActivity.java]: Can we have this as a bottomsheet guide like other instruments have?

Comment (harsh-2711) [reply app/src/main/java/org/fossasia/pslab/activity/AxisDetectionHelpActivity.java]: Sure.

**Comment by harsh-2711** [comment]
@CloudyPadmal I have made the changes and updated the apk. Please review it.

**Comment by harsh-2711** [comment]
@Avjeet I made a new layout because when I will implement external sensor support for HMC5883L then I have to add how to use it in this bottom sheet guide only. And so I have to change the layout as well.

**Pull request merged**